readme.md

```diff
@@ -82,7 +87,8 @@ Use ActiveRecord to create a new `pbj` sandwich and make it belong to the studen
 
 ```rb
 # Your answer...
-
+geraldo = students.create(name: "Geraldo")
+sandwiches.create(type: 'pbj', student: geraldo)
```

Comment:
Close.  What is assigned to your "sandwiches" variable?  This would cause an error (can not call "create" on nil).
`geraldo.sandwiches.create(type: 'obj')`